### PR TITLE
feat: show current project when installing helm chart (#613)

### LINKED ISSUE #613: As a user I dont know that the helm chart is installed in the current namespace
related to #590 

**Steps:**
1. EXEC: pick "Open Helm Charts" in ctx menu to the cluster in the resource tree
2. EXEC: in the list of helm charts, select a chart and hit "Install" in the details pane
3. EXEC: provide the name (and possibly additional parameters) and hit "Install"
4. EXEC: Answer the question: in what namespace/project was it installed?

**Result:**
Nothing in the dialog tells you to what namespace/project the chart is installed to. You can determine it though by looking at the resource tree where there's a child node for the current namespace. If you didnt expand the cluster node, you wont know what's the current namespace.

I'd add a row in the installation panel which tells you to what namespace the chart is installed to. This allows you to know beforehand and prevents you from installing to the wrong namespace/project

### PR DESCRIPTION
**Pull request by adietish**
fixes #613

### CODE REVIEW

File: src/main/java/org/jboss/tools/intellij/openshift/ui/helm/InstallPanel.java [hunk 1]
```diff
@@ -120,6 +123,11 @@ private void initComponents() {
     releaseNameText.addKeyListener(onKeyPressed(installButton));
     add(releaseNameText, "spanx 2, pushx, growx, wrap");
 
+    add(new JBLabel("Active Project:"),"skip");
```
Comment (sbouchet) [inline src/main/java/org/jboss/tools/intellij/openshift/ui/helm/InstallPanel.java:126]: Is it possible tochange that label given if the cluster is openshift (project) or kubernetes(namespace) ?

Comment (adietish) [reply src/main/java/org/jboss/tools/intellij/openshift/ui/helm/InstallPanel.java]: replaced by
```java
add(new JBLabel("Active " + (odo.isOpenShift()? "project:" : "namespace:")),"skip");
```

**Review by sbouchet — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
/override ci/prow/e2e-openshift  tide "Aggregated Test Report"

**Pull request merged**